platform/viewer/src/components/Snackbar/Snackbar.css

```diff
@@ -0,0 +1,235 @@
+.sb-container {
+  position: fixed;
+  width: 420px;
+  padding: 20px;
+  z-index: 9998;
+  box-sizing: border-box;
```

Comment:
I think we have a wildcard for this at the top level.

`* { box-sizing: border-box; }`, so you shouldn't need this.